packages/sdk/src/4.0/funcs.ts

```diff
@@ -6805,13 +6811,13 @@ export const search_groups = async (
  * GET /groups/search/with_roles -> IGroupSearch[]
  *
  * @param sdk IAPIMethods implementation
- * @param request composed interface "IRequestSearchGroups" for complex method parameters
+ * @param request composed interface "IRequestSearchGroupsWithRoles" for complex method parameters
  * @param options one-time API call overrides
  *
  */
 export const search_groups_with_roles = async (
   sdk: IAPIMethods,
-  request: IRequestSearchGroups,
+  request: IRequestSearchGroupsWithRoles,
```

Comment:
interesting. I wonder if we've introduced a breaking change in this SDK call.

Reply:
Not sure.

Reply:
This is just a revert so we will address when the sdk_23_18 actually rolls forward.